common/changes/scriby-dialog-on-dismissed_2017-01-31-19-40.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Dialog: Call onDismissed callback function when dialog close animation finishes.",
+      "type": "patch"
```

Comment:
Edit this (and the other change file) and change type to "minor". Minor fixes are non breaking, but introduce some new feature that is not available in previous patches.

Patches are for no prop change bug fixes. 

If the new param were not optional, or you were removing or significantly changing some behavior, that would be major. But I think you aren't doing that here.